ruby/tests/BUILD.bazel

```diff
@@ -65,7 +65,7 @@ sh_test(
     ],
     data = [
         "args_check.rb",
-        "@org_ruby_lang_ruby_toolchain//:ruby_bin",
+        "@ruby-system//:ruby_bin",
```

Comment:
`@ruby_system` seems more idiomatic for a repository name, and could maybe help distinguish between the repo name and the setting name? It's not 100% clear to me where this name is actually coming from.

Reply:
This is coming from sdk.bzl, which generates a repo for each toolchain and binds the system one to //external:rules_ruby_system_interpreter.

Note: I've updated this since your review.  It now looks like `@rules_ruby//ruby/runtime:interpreter`, and will select whichever ruby interpreter was resolved